continuous_integration/environment-3.12.yaml

```diff
@@ -61,11 +63,13 @@ dependencies:
   - lz4
   - psutil
   - requests
-  - scikit-image
+  # Temporarily removing to allow `numpy >=2` to be installed
+  # - scikit-image
```

Comment:
As of scikit-image 0.23.0, NumPy 2 is supported: https://github.com/scikit-image/scikit-image/issues/7282